src/tests/test_funds_api.py

```diff
@@ -1,11 +1,112 @@
+        assert {"fi": "Passive test fund 1 - fi, long", "sv": "Passive test fund 1 - en, long"} == fund["longName"]
+        assert {"fi": "Passive test fund 1 - fi, short", "sv": "Passive test fund 1 - en, short"} == fund["shortName"]
+        assert "#123456" == fund["color"] 
+        assert 1 == fund["risk"] 
+        assert {"fi": "PASSIVETESTFI", "sv": "PASSIVETESTSV"} == fund["KIID"]
+        assert "Passive test fund 1 - subs" == fund["bankReceiverName"]
+        assert "PASSIVE" == fund["group"]
+        assert "2021-11-10" == fund["priceDate"]
+        assert 1.2345 == fund["aShareValue"]
+        assert 1.098 == fund["bShareValue"]
+        assert 0.00 == fund["changeData"]["change1d"]
+        assert -0.01 == fund["changeData"]["change1m"]
+        assert -0.19 == fund["changeData"]["change1y"]
+        assert -0.27 == fund["changeData"]["change3y"]
+        assert -1.02 == fund["changeData"]["change5y"]
+        assert 1.22 == fund["changeData"]["change10y"]
+        assert 12.3 == fund["changeData"]["change15y"]
+        assert 25.5 == fund["changeData"]["change20y"]
+        assert -0.32 == fund["profitProjection"]
+        assert "2021-10-12" == fund["profitProjectionDate"]
+
+    def test_find_fund_invalid_id(self,
+                                  client: TestClient,
+                                  user_1_auth: BearerAuth
+                                  ):
+        for id in invalid_uuids:
+            url = "/v1/funds/{fund_id}".format(fund_id=id)
+            response = client.get(url, auth=user_1_auth)
+            assert response.status_code == 400
+
+    def test_list_funds(self, client: TestClient, user_1_auth: BearerAuth):
+        response = client.get("/v1/funds", auth=user_1_auth)
+        assert response.status_code == 200
+
+        response_funds = response.json()
+        response_ids = list(map(lambda i: i["id"], response_funds))
+        assert 6 == len(response_funds)
+        for fund_id in fund_ids.values():
+            assert fund_id in response_ids
+
+    def test_list_funds_limits(self,
+                               client: TestClient,
+                               user_1_auth: BearerAuth
+                               ):
+        self.assert_list_count(
+            client=client,
+            auth=user_1_auth,
+            expected_count=3,
+            first_result=0,
+            max_results=3
+        )
+        """
+        self.assert_list_count(
+            client=client,
+            auth=user_1_auth,
+            expected_count=4,
+            first_result=2,
+            max_results=8
+        )
+        """
+
+    def assert_list_count(self,
+                          expected_count: int,
+                          client: TestClient,
+                          auth: BearerAuth,
+                          first_result: int,
+                          max_results: int
+                          ):
+        query = "first_result={first_result}&max_results={max_results}".format(
```

Comment:
you can change this one too:
        query = "first_result={first_result}&max_results={max_results}".format(
            first_result=first_result,
            max_results=max_results
        ) 
        ->
               query = f"first_result={first_result}&max_results={max_results}"